tests/run_tests.sh

```diff
@@ -25,6 +25,12 @@ function print_graphics_data () {
 }
 
 
+
+function detect_known_issues () {
```

Comment:
Should we distinguish known issues from _retryable_ issues?

Reply:
Yes, we should. I renamed those to retryable issues.

I believe tests should still fail on known unretryable issues, so I didn't add anything specific for those.